Plugins/Public/autobuy/Main.cpp

```diff
@@ -288,21 +315,38 @@ void AutobuyInfo(uint iClientID)
 	PrintUserCmdText(iClientID, L"Error: Invalid parameters");
 	PrintUserCmdText(iClientID, L"Usage: /autobuy <param> [<on/off>]");
 	PrintUserCmdText(iClientID, L"<Param>:");
-	PrintUserCmdText(iClientID, L"   info - display current autobuy-settings");
-	PrintUserCmdText(iClientID, L"   missiles - enable/disable autobuy for missiles");
-	PrintUserCmdText(iClientID, L"   torps - enable/disable autobuy for torpedos");
-	PrintUserCmdText(iClientID, L"   mines - enable/disable autobuy for mines");
-	PrintUserCmdText(iClientID, L"   cd - enable/disable autobuy for cruise disruptors");
-	PrintUserCmdText(iClientID, L"   cm - enable/disable autobuy for countermeasures");
-	PrintUserCmdText(iClientID, L"   reload - enable/disable autobuy for nanobots/shield batteries");
-	PrintUserCmdText(iClientID, L"   repair - enable/disable auto-repair");
-	PrintUserCmdText(iClientID, L"   all: enable/disable autobuy for all of the above");
+	PrintUserCmdText(iClientID, L"info - display current autobuy-settings");
+	PrintUserCmdText(iClientID, L"missiles - enable/disable autobuy for missiles");
+	PrintUserCmdText(iClientID, L"torps - enable/disable autobuy for torpedos");
+	PrintUserCmdText(iClientID, L"mines - enable/disable autobuy for mines");
+	PrintUserCmdText(iClientID, L"cd - enable/disable autobuy for cruise disruptors");
+	PrintUserCmdText(iClientID, L"cm - enable/disable autobuy for countermeasures");
+	PrintUserCmdText(iClientID, L"bb - enable/disable autobuy for nanobots/shield batteries");
+	PrintUserCmdText(iClientID, L"munition - enable/disable autobuy for ammo");
+	PrintUserCmdText(iClientID, L"cloak - enable/disable autobuy for cloak batteries");
+	PrintUserCmdText(iClientID, L"jump - enable/disable autobuy for jump drive batteries");
+	PrintUserCmdText(iClientID, L"matrix - enable/disable autobuy for hyperspace matrix batteries");
+	PrintUserCmdText(iClientID, L"repair - enable/disable auto-repair");
+	PrintUserCmdText(iClientID, L"all: enable/disable autobuy for all of the above");
 	PrintUserCmdText(iClientID, L"Examples:");
 	PrintUserCmdText(iClientID, L"\"/autobuy missiles on\" enable autobuy for missiles");
 	PrintUserCmdText(iClientID, L"\"/autobuy all off\" completely disable autobuy");
 	PrintUserCmdText(iClientID, L"\"/autobuy info\" show autobuy info");
 }
-
+void UpdatedStatusList(uint iClientID)
+{
+	PrintUserCmdText(iClientID, L"%s : Missiles (missiles)", mapAutobuyPlayerInfo[iClientID].bAutoBuyMissiles ? L"ON" : L"OFF");
```

Comment:
the status before the type? that feels backwards

Reply:
I did it because statuses ON/OFF go in one properly aligned column
easier to see which are ON and OFF.

Reply:
could've done it properly aligned with spaces the other way around though right?

Reply:
I could...

but this placement is easier to track for eye
L"%s : Missiles (missiles)
L"%s : Mines (mines)
L"%s : Torpedos (torps)
L"%s : Hyperspace Matrix Batteries (matrix)

while this placement is harder to align properly because of small every symbol different length
+big space to values from string makes it harder to track than my current version
thi
Missiles (missiles)                                 L"%s : 
Mines (mines)                                      L"%s : 
Torpedos (torps)                                  L"%s : 
Hyperspace Matrix Batteries (matrix)   L"%s :